tests/test_embeds.py

```diff
@@ -423,3 +423,17 @@ def test_copy_empty() -> None:
     e = Embed.from_dict({})
     copy = e.copy()
     assert e.to_dict() == copy.to_dict() == {}
+
+
+# backwards compatibility
+def test_emptyembed() -> None:
+    with pytest.warns(DeprecationWarning):
+        assert embeds.EmptyEmbed is None  # type: ignore
+    with pytest.warns(DeprecationWarning):
+        assert Embed.Empty is None  # type: ignore
+    with pytest.warns(DeprecationWarning):
+        assert Embed().Empty is None  # type: ignore
```

Comment:
we should run these twice to make sure they throw a second warning when accessed a second time.

Reply:
`pytest.warns` uses `warnings.simplefilter("always")`, so we unfortunately can't really test that.